easy-ui-react/src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,83 @@
+@use "../styles/common" as *;
+
+// prettier-ignore
+.statusPromotional {
+  @include component-token("notification", "color", theme-token("color.text.inverse"));
+  @include component-token("notification", "color.background", theme-token("color.background.primary"));
+}
+
+// prettier-ignore
+.statusSuccess {
+  @include component-token("notification", "color", theme-token("color.text.success.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.success.subtle"));
+}
+
+// prettier-ignore
+.statusNeutral {
+  @include component-token("notification", "color", theme-token("color.text.secondary.bold"));
+  @include component-token("notification", "color.background",theme-token("color.background.secondary.subtle"));
+}
+
+// prettier-ignore
+.statusWarning {
+  @include component-token("notification", "color", theme-token("color.text.inverse.inverse.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.warning.subtle"));
+}
+
+// prettier-ignore
+.statusError {
+  @include component-token("notification", "color", theme-token("color.text.danger.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.danger.subtle"));
+}
+
+.typeToast {
+  display: inline-flex;
+}
+
+.typeAlert {
+  display: flex;
+  width: 100%;
+}
+
+// prettier-ignore
+.Notification {
+  @include component-token("notification", "border_radius", design-token("shape.border_radius.base"));
+  @include component-token("notification", "max-width", 1200px);
+  align-items: center;
+  justify-content: space-between;
+  margin: auto;
+  padding: design-token("space.1");
+  border-radius: 0 0 component-token("notification", "border_radius") component-token("notification", "border_radius");
+  max-width: component-token("notification", "max-width");
+  color: component-token("notification", "color");
+  background-color: component-token("notification", "color.background");
+}
+
+.iconText {
+  display: inline-flex;
+  align-items: center;
+  gap: design-token("space.1");
+}
+
+// prettier-ignore
+.closeButton {
+  border: 0;
+  padding: design-token("space.0");
+  margin: design-token("space.0.5") design-token("space.1") design-token("space.0.5") design-token("space.2");
+  background-color: component-token("notification", "color.background");
+  color: component-token("notification", "color.text");
+  cursor: pointer;
+}
+
+.notificationRegion {
```

Comment:
**nit:** since notification is the name of the component would it make sense for these to be more brief and just be `region` and `container`?

Reply:
i think this is a good call. i also added `absolute` positioning to the container

Reply:
the problem with using `absolute` on an element that's connected to `document.body` is that it will move with the scroll, so if a user is scrolled down on the page and a toast appears it may be above what's visible on their screen. so `fixed` position is the better positioning to use when attached to the body.

`absolute` positioning would work if it was a child of the sticky header in our easypost-web app but that would require the consumer to be in control of the container. i guess one option would be allowing for a custom portal container to be passed in along with "fixed" or "absolute" being passed in to the `notificationPlacementStyle`, etc

Reply:
if i understand the suggestion correctly, i refactored the logic a bit for the `Provider` to accept a generic `notificationPlacementProps` prop which is an object that can accept an `htmlId`, `positionType`, and `positionPlacement` prop. if `htmlId` is provided, then we render the notifications into that container instead of the `document.body`

```tsx
export type NotificationPositionPlacement = {
  /** Top offset */
  top?: string;
  /** Right offset */
  right?: string;
  /** Bottom offset */
  bottom?: string;
  /** Left offset */
  left?: string;
};

export type NotificationPositionType = "fixed" | "absolute";

export type NotificationPlacementProps = {
  /** HTML ID of element where notifications will render to */
  htmlId?: string;
  /** Position type */
  positionType?: NotificationPositionType;
  /** Position placement */
  positionPlacement?: NotificationPositionPlacement;
};

```